### DISCUSSION TITLE
视频bv号自动转换重定向在新版本中不存在了

### DISCUSSION THREAD

**Discussion by hstdt** [the1812/Bilibili-Evolved#2631] [功能建议 / Ideas]
老版本能自动重定向到av号链接上，但是2.X新版本貌似只能在功能里面多点一步。能够在BV 号转换组件里面增加一个自动重定向开关嘛。

**Comment by prajna0** [comment]
我也想要

**Comment by Geeyun-JY3** [comment]
在 v1 这是个功能，是你们之前开过，后面可视化设置开关在 v1 就已经隐藏了：https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.12
不过当时是留有控制台开关的。

**Reply by prajna0** [reply]
在控制台中输入bilibiliEvolved.settings.preferAvUrl = true
没有开启成功…

**Reply by Geeyun-JY3** [reply]
没说 v2 这个开关还能用，给你们补充一下信息而已。

**Comment by the1812** [comment]
~~对哦, v1 的隐藏功能, 我好像没做迁移(~~
原来是迁移了, 但还有 bug, 修复后会放出.
顺便还发现现在 Chrome 会同时记录转换前后的地址了, 搞得历史记录很多重复的标题 (网址参数清理也是)
